### PR TITLE
Create group function

### PR DESCRIPTION
**Pull request by frmad**
I need help with the post method

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -102,6 +102,10 @@ compose.experimental {
 libres {
 }
 tasks.getByPath("jsProcessResources").dependsOn("libresGenerateResources")
+dependencies {
+    implementation(libs.androidx.ui.tooling.preview.android)
```
Comment (mathiasgredal) [inline app/build.gradle.kts:106]: What is this used for?

File: app/src/commonMain/kotlin/sdu/mobile/xpence/ui/components/createGroup/CreateGroup.kt [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            Text(text = "Create New Group")
+        }
+    }
+
+    if (isDialogVisible) {
+        createDialog(
+            onDismiss = { isDialogVisible = false },
+        )
+    }
+
```
Comment (mathiasgredal) [inline app/src/commonMain/kotlin/sdu/mobile/xpence/ui/components/createGroup/CreateGroup.kt:47]: Blank lines

File: app/src/commonMain/kotlin/sdu/mobile/xpence/ui/components/createGroup/CreateGroup.kt [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    onDismiss: () -> Unit
+) {
+    val coroutineScope = rememberCoroutineScope()
+
+    Dialog(
+        onDismissRequest = { onDismiss() },
+        properties = DialogProperties(dismissOnClickOutside = false)
+    ) {
+        Column(
+            modifier = Modifier
```
Comment (mathiasgredal) [inline app/src/commonMain/kotlin/sdu/mobile/xpence/ui/components/createGroup/CreateGroup.kt:63]: You shouldn’t need full Max height and width after calling fillmaxsize

File: app/src/commonMain/kotlin/sdu/mobile/xpence/ui/components/createGroup/CreateGroup.kt [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+            Spacer(modifier = Modifier.height(5.dp))
+
+            groupDescriptionTextField(description = description, onTextChange = { description = it })
+
+            Spacer(modifier = Modifier.height(10.dp))
+
+            var selected by remember { mutableStateOf(listOf<User>()) }
+
+            val current by usingAPI { client ->
+                getCurrentUser(client)
```
Comment (mathiasgredal) [inline app/src/commonMain/kotlin/sdu/mobile/xpence/ui/components/createGroup/CreateGroup.kt:89]: You can combine these 2 api request

File: gradle/libs.versions.toml [hunk 5]
```diff
@@ -39,6 +39,8 @@ ktor-json = { module = "io.ktor:ktor-serialization-kotlinx-json", version.ref =
 ktor-content-negotiation = { module = "io.ktor:ktor-client-content-negotiation", version.ref = "ktor" }
 multiplatform-settings = { module = "com.russhwolf:multiplatform-settings", version.ref = "multiplatform-settings" }
 multiplatform-settings-noarg = { module = "com.russhwolf:multiplatform-settings-no-arg", version.ref = "multiplatform-settings" }
+androidx-ui-tooling-preview-android = { group = "androidx.compose.ui", name = "ui-tooling-preview-android", version = "1.5.4" }
```
Comment (mathiasgredal) [inline gradle/libs.versions.toml:42]: What is this used for

**Review by mathiasgredal — CHANGES_REQUESTED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (frmad) [reply app/build.gradle.kts:106]: preview in android studio

**Review by mathiasgredal — APPROVED** [review]

**Pull request merged**